.env.sample

```
@@ -18,4 +18,5 @@ BACKEND_CONTAINER_NAME=fastify-starter-backend
 BACKEND_LOCAL_PORT=3000
 DB_CONTAINER_NAME=fastify-starter-db
 DB_LOCAL_PORT=3000
-GRAFANA_WHITELIST=127.0.0.1,127.0.0.2
+METRICS_IP_WHITELIST=127.0.0.1,127.0.0.2
+LOKI_LOGGING=false;
```

```suggestion
WINSTON_LOKI_URL=http://localhost:3000;
```
If not set don't add transport